src/commands/git-metadata/upload.ts

```diff
@@ -213,5 +213,5 @@ UploadCommand.addPath('git-metadata', 'upload')
 UploadCommand.addOption('dryRun', Command.Boolean('--dry-run'))
 UploadCommand.addOption('verbose', Command.Boolean('--verbose'))
 UploadCommand.addOption('directory', Command.String('--directory'))
-UploadCommand.addOption('gitSync', Command.Boolean('--git-sync'))
+UploadCommand.addOption('noGitSync', Command.Boolean('--no-git-sync'))
```

Comment:
Just a clarification, shouldn't we think about backwards compatibility? What if customers are already using this command line option, and after we remove it their pipeline breaks because datadog-ci failed on unknown option? Do we have some kind of policy on this?

Reply:
Yes indeed, thanks! I'll fix it.